packages/payment-proxy-client/src/constants.ts

```diff
@@ -2,7 +2,7 @@
 export const QUERY_KEY = "rpcUrl";
 export const costPerByte = 1;
 
-export const proxyUrl = "https://www.w3.org";
+export const proxyUrl = import.meta.env.VITE_PROXY_URL;
```

Comment:
Are we setting this anywhere?

Reply:
That is a good point, fixed in https://github.com/statechannels/go-nitro/pull/1735/commits! I'm surprised linting doesn't catch this, since we do define our env var names in a typing file.